app/src/main/java/com/sopt/dive/feature/login/LoginScreen.kt

```diff
@@ -88,51 +87,38 @@ fun LoginScreen(
         }
 
         OutlinedTextField(
-            value = idText,
-            onValueChange = { idText = it },
+            value = uiState.userId,
+            onValueChange = { viewModel.updateUserId(it) },
             modifier = Modifier.fillMaxWidth(),
             textStyle = TextStyle(brush = brush),
             label = { Text("아이디를 입력하세요") },
-            keyboardOptions = KeyboardOptions(
-                imeAction = ImeAction.Next
-            ),
-            keyboardActions = KeyboardActions(
-                onNext = {
-                    focusRequester.requestFocus()
-                }
-            ),
+            keyboardOptions = KeyboardOptions(imeAction = ImeAction.Next),
+            keyboardActions = KeyboardActions(onNext = { focusRequester.requestFocus() }),
             singleLine = true
         )
 
         Spacer(modifier = Modifier.height(24.dp))
-
         Text(text = "PW", fontSize = 20.sp)
 
         OutlinedTextField(
-            value = pwText,
-            onValueChange = { pwText = it },
+            value = uiState.password,
+            onValueChange = { viewModel.updatePassword(it) },
```

Comment:
이것도
```suggestion
            onValueChange = viewModel::updatePassword,
```
로 써볼 수 있음! 안두콩 꿀팁 공유에 자료 적어두었습니다:)